extensions/shared/paid-blocks/utils.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { compact, get, startsWith } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { isSimpleSite } from '../site-type-utils';
+import getJetpackExtensionAvailability from '../get-jetpack-extension-availability';
+import getSiteFragment from "../get-site-fragment";
+
+/*
+ * Blocks list that require a paid plan.
+ */
+export const PAID_BLOCKS_LIST = [
```

Comment:
It doesn't look like this constant is being used outside of `isUpgradable`. Are you expecting it will be or should we move it inside that function?

Reply:
Personally I feel this would be best in a separate _file_ within `shared/paid-blocks` so that anyone looking for the definition would find it easily. Perhaps

`shared/paid-block/constants.js`

?